app/views/blog_posts/show.rb

```diff
@@ -3,7 +3,7 @@
 module BlogPosts
   class Show < ApplicationView
     def blog_title
-      presenter.title
+      [presenter.title, t('site.blogs.list.page-title')]
```

Comment:
The return value of this method is used for the *on-page* title of the blog post. This needs to be a single string, i.e. the return value of `presenter.title`.